api/src/main/java/com/velocitypowered/api/proxy/ProxyServer.java

```diff
@@ -17,13 +17,14 @@
 import java.util.Collection;
 import java.util.Optional;
 import java.util.UUID;
-import net.kyori.text.Component;
+import net.kyori.adventure.audience.MultiAudience;
+import net.kyori.adventure.text.Component;
 import org.checkerframework.checker.nullness.qual.NonNull;
 
 /**
  * Provides an interface to a Minecraft server proxy.
  */
-public interface ProxyServer {
+public interface ProxyServer extends MultiAudience {
```

Comment:
I would recommend extending the more generic `Audience` in the `api` interfaces, then using `MultiAudience` in the implementation (for `ProxyServer` and co.) That way you don't expose `audiences()` in the spec.